modules/openapi-generator/src/main/resources/Java/libraries/microprofile/model.mustache

```diff
@@ -37,9 +37,12 @@ import {{rootJavaEEPackage}}.json.bind.serializer.SerializationContext;
 import {{rootJavaEEPackage}}.json.stream.JsonGenerator;
 import {{rootJavaEEPackage}}.json.stream.JsonParser;
 import {{rootJavaEEPackage}}.json.bind.annotation.JsonbProperty;
-{{#vendorExtensions.x-has-readonly-properties}}
+{{#jsonbPolymorphism}}
+import {{rootJavaEEPackage}}.json.bind.annotation.JsonbSubtype;
+import {{rootJavaEEPackage}}.json.bind.annotation.JsonbTransient;
+import {{rootJavaEEPackage}}.json.bind.annotation.JsonbTypeInfo;
+{{/jsonbPolymorphism}}
 import {{rootJavaEEPackage}}.json.bind.annotation.JsonbCreator;
```

Comment:
The JsonbCreator import was previously never included, the `vendorExtensions.x-has-readonly-properties` is only available in the `#model` context I think. For Jackson the import only work because it is added in the `#imports`.